packages/application/src/path.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class Path {
+  /**
+   * Returns the directory name of a file path
+   *
+   * @param path a file path as in '/foo/bar/baz/asdf/file.ext'
+   * Returns: '/foo/bar/baz/asdf'
```

Comment:
separate from my opinions on this util, this doc is incorrect. this return includes a trailing `/`